src/utils/daos_autotest.c

```diff
@@ -27,8 +27,26 @@
 /** Input arguments passed to daos utility */
 struct cmd_args_s *autotest_ap;
 
+/* The start and end time in ns for individual test*/
+uint64_t	t_start, t_end;
+char		iops[32];
 /** How many concurrent I/O in flight */
 #define MAX_INFLIGHT 16
+#define OUTPUT_IOPS(x) ({\
+		/* negative value is unexpected*/\
+		if (x < 0.0) {\
```

Comment:
Could be better to print an error for such unexpected negative value

Reply:
Just updated to print a warning message for unexpected value.